jsonschema/current/Interfaces.schema.json

```diff
@@ -97,6 +90,18 @@
             "oneOf": [
                 {"$ref": "dungeoneer/statuseffects/StatusEffect.schema.json" }
             ]
+        },
+        "TriggerLike": {
+            "oneOf": [
+                {"$ref": "dungeoneer/entities/triggers/BasicTrigger.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/ButtonModel.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/ButtonDecal.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/ConditionalTrigger.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/DamageTrigger.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/LevelSwap.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/LookAtTrigger.schema.json"},
+                {"$ref": "dungeoneer/entities/triggers/Trigger.schema.json"}
```

Comment:
The reason I did not do this is because there is no single Trigger base class. There are strangely two:
- `Trigger`
- `BasicTrigger`

So this schema interface cannot be used.

Reply:
That is to say this is invalid:
```json
"useTrigger": {
    "$ref": "../../Interfaces.schema.json#/definitions/TriggerLike",
    "description": "Trigger to target for use action.",
    "baseClass": "Actor"
}
```

Which I left in `Actor.schema.json` 🤦

Reply:
We technically should have:
`TriggerLike` and `BasicTriggerLike` to account for the inheritance paths.

Reply:
Yeah, this is the reason I added it. I think we already had this discussion? I just forgot^^

Reply:
Alright, reverted the `TriggerLike` changes.